lms/static/scripts/frontend_apps/components/dashboard/GradeStatusChip.tsx

```diff
@@ -8,6 +8,19 @@ export type GradeStatusChipProps = {
   grade: number;
 };
 
+function formatGrade(grade: number): string {
+  /**
+   * Format a grade from 0 to 1 to a string. If the grade is an integer, it
+   * will be returned as an integer string. Otherwise, it will be returned
+   * with two decimal places.
+   */
```

Comment:
In JS the JSDoc is defined over the function definition, not below 😅 

```suggestion
/**
 * Format a grade from 0 to 1 to a string. If the grade is an integer, it
 * will be returned as an integer string. Otherwise, it will be returned
 * with two decimal places.
 */
function formatGrade(grade: number): string {
```